lib/resources/nginx_conf.rb

```diff
@@ -82,10 +82,13 @@ def resolve_references(data, rel_path)
       # Any call to `include` gets its data read, parsed, and merged back
       # into the current data structure
       if data.key?('include')
-        data.delete('include').flatten
-            .map { |x| File.expand_path(x, rel_path) }
-            .map { |path| parse_nginx(path) }
-            .map { |e| data.merge!(e) }
+        include_paths = data.delete('include').flatten.map do |path|
+          paths = inspec.command("ls #{File.expand_path(path, rel_path)}").stdout.split
```

Comment:
Thank you @arlimus , I replaced the ls command with Findfiles library...looks like Findfiles does not work with Windows either. Looks like updating Findfiles to handle Windows too will be more useful overall.

PS. Looks like the PR is failing AppVeyor version 21 with the message:
An error occurred while installing mixlib-shellout (2.3.1), and Bundler cannot continue